metadata-ingestion/src/datahub/api/entities/platformresource/platform_resource.py

```diff
@@ -69,71 +76,75 @@ def to_resource_info(self) -> models.PlatformResourceInfoClass:
         )
 
 
-class OpenAPIGraphClient:
+class DataPlatformInstanceUrn:
+    """
+    A simple implementation of a URN class for DataPlatformInstance.
+    Since this is not present in the URN registry, we need to implement it here.
```

Comment:
The urn types are autogenerated- we can definitely just add this there instead of building a new type